### PR TITLE
[14.0][IMP] account_chart_update: Update account groups too

### PR DESCRIPTION
**Pull request by etobella**
With this change, we are able to update the account groups

### CODE REVIEW

**Comment by rafaelbn** [comment]
@moduon MT-2594

File: account_chart_update/wizard/wizard_chart_update.py [hunk 1]
```diff
@@ -557,6 +616,42 @@ def find_account_by_templates(self, templates):
...
+        account_model = self.env["account.group"]
+        for matching in self.account_group_matching_ids.sorted("sequence"):
+            if matching.matching_value == "xml_id":
+                real = self.env["account.group"]
+                for template in templates:
+                    try:
+                        real |= self.env.ref(self._get_real_xml_name(template))
+                    except BaseException:
+                        pass
+
```
Comment (pedrobaeza) [inline account_chart_update/wizard/wizard_chart_update.py]: No need of this empty line

File: account_chart_update/wizard/wizard_chart_update.py [hunk 2]
```diff
@@ -557,6 +616,42 @@ def find_account_by_templates(self, templates):
...
+                if not codes:
+                    continue
+                criteria = ("code", "in", list(map(self.padded_code, codes)))
+            else:
+                field_name = matching.matching_value
+                field_values = templates.mapped(field_name)
+                if not field_values:
+                    continue
+                criteria = (field_name, "in", field_values)
+
```
Comment (pedrobaeza) [inline account_chart_update/wizard/wizard_chart_update.py]: No empty line

File: account_chart_update/wizard/wizard_chart_update.py [hunk 3]
```diff
@@ -557,6 +616,42 @@ def find_account_by_templates(self, templates):
...
+            result = account_model.search(
+                [criteria, ("company_id", "=", self.company_id.id)]
+            )
+            if result:
+                return result.id
+
```
Comment (pedrobaeza) [inline account_chart_update/wizard/wizard_chart_update.py]: No empty line

**Comment by etobella** [comment]
BTW, I choose as parameter the code_prefix_start, due to the following constrain:

https://github.com/odoo/odoo/blob/14.0/addons/account/models/account_account.py#L570

**Review by pedrobaeza — APPROVED** [review]

**Review by Shide — APPROVED** [review]
Functional review

**Pull request merged**